functions/openhabian.bash

```diff
@@ -149,55 +149,65 @@ openhabian_update_check() {
-        introText="You are currently using the \"${current}\" openHABian environment version. I will ONLY work with openHAB version 2.\\nIf you want to run openHAB 3 you need to switch to the \"openHAB3\" branch. If that's your intent better let the upgrade function do it for you and validate afterwards you are on openHAB3 branch.\\n\\nThe very latest openHAB 2 version is called \"master\".\\nThis is providing you with the latest (openHAB2 !) features but less people have tested it so it is a little more likely that you run into errors.\\nYou can step back a little and switch to use the stable version now.\\nYou can switch at any time by selecting this menu option again or by setting the 'clonebranch=' parameter in '/etc/openhabian.conf'.\\n"
+        introText="You are currently using the \"${current}\" openHABian environment version. I will ONLY work with openHAB version 2.\\nIf you want to run openHAB 3 you need to switch to the \"openHAB3\" branch. If that's your intent better let the upgrade function do it for you and validate afterwards you are on main branch.\\n\\nThe very latest openHAB 2 version is called \"master\".\\nThis is providing you with the latest (openHAB2 !) features but less people have tested it so it is a little more likely that you run into errors.\\nYou can step back a little and switch to use the stable version now.\\nYou can switch at any time by selecting this menu option again or by setting the 'clonebranch=' parameter in '/etc/openhabian.conf'.\\n"
     else
-      if [[ $current == "openHAB3" ]]; then
-          introText="You are currently the latest (\"openHAB3\") openHABian environment version.\\nYou will need to be on here (openHAB3) if you want to run openHAB version 3.\\n\\nYou can switch environments at any time by selecting this menu option again or by setting the 'clonebranch=' parameter in '/etc/openhabian.conf'.\\n\\nATTENTION: If you want to upgrade to openHAB 3 you should select \"Cancel\" now and select option 03 from the main menu instead."
+      if [[ $current == "main" ]] || [[ $current == "openHAB3" ]]; then
+          introText="You are currently using the \"${current}\" openHABian environment version.\\n\\nThe very latest openHAB 3 version is called \"main\".\\nThis is providing you with the latest (openHAB3 !) features but less people have tested it so it is a little more likely that you run into errors.\\nYou can step back a little and switch to use the stable version now called \"openHAB3\".\\nYou can switch at any time by selecting this menu option again or by setting the 'clonebranch=' parameter in '/etc/openhabian.conf'.\\n"
       else
           introText="You are currently using an unknown branch of openHABian.\\nThis may be a test version or an error, if so please report on Github (remember to provide a debug log - see debug guide)."
       fi
     fi
 
     if [[ $current == "stable" ]]; then
-      stable="ON"
+      main="OFF"
```

Comment:
Compact this code by setting all to "off" first then only change the variable to represent the "current" branch

Reply:
That makes sense. This is why we review to fix our stupid mistakes.